cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -645,11 +639,11 @@ pub enum CoreEnvironmentError {
     #[error("failed to upgrade environment")]
     UpgradeFailed(#[source] CallPkgDbError),
     // endregion
-    #[error("call to `pkgdb buildenv` failed")]
-    BuildEnvCall(#[source] std::io::Error),
+    #[error("error building environment")]
+    BuildEnv(#[source] CallPkgDbError),
 
-    #[error("error building environment: {0}")]
-    BuildEnv(String),
+    #[error("unexpected output from pkgdb buildenv")]
```

Comment:
Suggestion: since this is going to be end-user facing we probably shouldn't mention the subcommands or subsystems we're using. I think a more appropriate error would be something similar to `unexpected output from environment builder command`.

Reply:
Fixing